app/stores/CommentsStore.ts

```diff
@@ -28,14 +32,29 @@ export default class CommentsStore extends Store<Comment> {
    * @param documentId ID of the document to get comments for
    * @returns Array of comments
    */
-  threadsInDocument(documentId: string): Comment[] {
-    return this.filter(
+  threadsInDocument(
+    documentId: string,
+    options: CommentSortOption = { type: CommentSortType.Chrono }
+  ) {
+    const comments = this.filter(
       (comment: Comment) =>
         comment.documentId === documentId &&
         !comment.parentCommentId &&
         (!comment.isNew ||
           comment.createdById === this.rootStore.auth.currentUserId)
     );
+
+    if (options.type === CommentSortType.Chrono) {
+      return comments;
```

Comment:
We should delete the overridden method here, as it is the `MostRecent` sort.
https://github.com/outline/outline/blob/e41519575f4ad1b4ee7e429b18feddc8f9ed4884/app/stores/CommentsStore.ts#L142-L144